qutebrowser/config/parsers/keyconf.py

```diff
@@ -142,9 +142,14 @@ def _str_section_desc(self, sectname):
     def save(self):
         """Save the key config file."""
         log.destroy.debug("Saving key config to {}".format(self._configfile))
-        with qtutils.savefile_open(self._configfile, encoding='utf-8') as f:
-            data = str(self)
-            f.write(data)
+
+        try:
+            with qtutils.savefile_open(self._configfile,
+                                       encoding='utf-8') as f:
+                data = str(self)
+                f.write(data)
+        except (OSError, qtutils.QtOSError) as e:
```

Comment:
`QtOSError` inherits `OSError`, so I don't think it's needed to list them both here.